### ISSUE TITLE
Automatic publication of the package

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-rust#9]
### What
- Automate the publication of the package

**Comment by abughalib** [comment]
Can I work on this?

**Comment by teolemon** [comment]
yes of course, assigning you

**Comment by manishtiwari25** [comment]
I don't think we have to do anything on this. we can raise a PR and release please should take care of rest. 
https://github.com/manishtiwari25/openfoodfacts-rust/actions/runs/5045695165/jobs/9050552805

**Comment by ThiefMaster** [comment]
It would be great if this crate was published to crates.io instead of having to install it from Git.
